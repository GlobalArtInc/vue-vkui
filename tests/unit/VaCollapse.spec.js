import { shallowMount } from '@vue/test-utils'
import VaCollapse from '@/Collapse/VaCollapse.vue'
import VaCollapsePanel from '@/Collapse/VaCollapsePanel.vue'

describe('VaCollapse.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount({
      render () {
        return (
          <VaCollapse>
            <VaCollapsePanel header="header">body</VaCollapsePanel>
            <VaCollapsePanel header="header">body</VaCollapsePanel>
            <VaCollapsePanel header="header">body</VaCollapsePanel>
          </VaCollapse>
        )
      }
    })
    expect(wrapper).toMatchSnapshot()
  })
})
