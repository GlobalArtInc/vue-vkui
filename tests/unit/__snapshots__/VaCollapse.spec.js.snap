// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VaCollapse.vue renders correctly 1`] = `
<div class="va-collapse">
  <div class="va-collapse__panel">
    <div class="va-collapse__panel__header">
      <va-icon-stub type="angle-right"></va-icon-stub>
      header
    </div>
    <va-collapse-transition-stub>
      <!---->
    </va-collapse-transition-stub>
  </div>
  <div class="va-collapse__panel">
    <div class="va-collapse__panel__header">
      <va-icon-stub type="angle-right"></va-icon-stub>
      header
    </div>
    <va-collapse-transition-stub>
      <!---->
    </va-collapse-transition-stub>
  </div>
  <div class="va-collapse__panel">
    <div class="va-collapse__panel__header">
      <va-icon-stub type="angle-right"></va-icon-stub>
      header
    </div>
    <va-collapse-transition-stub>
      <!---->
    </va-collapse-transition-stub>
  </div>
</div>
`;
